docs/source/devguides/walkthroughs/DrivenMatlabAgent-Walkthrough.rst

```diff
@@ -72,35 +75,118 @@ Installation:
 
     Follow link: http://volttron.readthedocs.io/en/develop/install.html
 
-2. Add subtree volttron-applications under volttron/applications by using
-   the following command:
+2. Clone volttron-applications in any directory outside of your volttron
+   source directory. (Best practise: clone in the parent directory of volttron
+   source. For example, /users/user1/volttron and
+   /users/user1/volttron-applications):
 
 .. code-block:: console
 
-    git subtree add --prefix applications https://github.com/VOLTTRON/volttron-applications.git develop --squash
+    git clone https://github.com/VOLTTRON/volttron-applications.git
 
-Configuration
-~~~~~~~~~~~~~
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+To test the interaction between matlab driven agent and matlab code you would
+ need the following agents installed and running
+
+1. Run master driver and configure it to publish device data. Data published
+   by the device will be scraped by master driver agent and published to message
+   bus. DrivenMatlabAgent subscribes to device topics and sends the data to
+   matalab code.
+
+   The easiest way to do this for testing is to use the volttron-cfg command.
```

Comment:
get rid of the easiest way to do this...

To test the interaction between the masterdriver and matlab agent install a fake driver using 
vcfg --agents master_driver listener
command. 

put the use this command when platform is running in a note.